docs/document/docs/components/mobile/mobile-textarea.md

```diff
@@ -8,10 +8,9 @@ sidebar_label: MobileTextArea
 
 The MobileTextArea component allows the user to display multiple lines of text element.
 
-<div class="sample-container display-block-mobile" id="mobile-textarea">
-  <div id="sample-container__components" class="mobile"></div>
-</div>
-<script src="/js/samples/mobile/mobile-textarea.js"></script>
+import { MobileTextAreaComponent } from "@site/static/js/samples/mobile/textarea.js"
+
+<MobileTextAreaComponent />
```

Comment:
The default width setting seems wrong.
When resizing the component, it will be expanded both left and right direction. Please only resize with right direction.

Please check the previous one.

Reply:
i have fixed

Reply:
It seems the component preview position is a bit right.
Please modify the position.

Reply:
i have fixed

Reply:
The position issue has been fixed, but the issue here appeared again.
https://github.com/kintone-labs/kintone-ui-component/pull/1363#discussion_r1241969428

Reply:
Sorry about that, i have fixed once again

Reply:
Confirmed.